packages/create-app/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@backstage/create-app",
+  "description": "Create app package for Backstage",
+  "version": "0.1.1",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/spotify/backstage",
+    "directory": "packages/create-app"
+  },
+  "keywords": [
+    "backstage"
+  ],
+  "license": "Apache-2.0",
+  "main": "src/index.ts",
+  "bin": {
+    "backstage-create-app": "bin/backstage-create-app"
+  },
+  "dependencies": {
+    "commander": "^4.1.1",
+    "chalk": "^4.0.0",
+    "fs-extra": "^9.0.0",
+    "react-dev-utils": "^10.2.1",
```

Comment:
I think we can get rid of this right? Afaik it's only used in the bundling parts of the cli.